examples/analytics-iframe-transport-remote-frame.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Requests Frame</title>
+    <script>
+      window.addEventListener('amp-iframeTransportClientCreated', function() {
```

Comment:
Agree with this, too. Will leave myself a note to also change https://github.com/ampproject/amphtml/blob/d98d8e1dff2b26ebf773194384733af4e6415703/examples/ampcontext-creative.html#L12 after fixit week.